### ISSUE TITLE
style the upload page

### ISSUE THREAD

**Issue by AJM-1** [computationalmystic/RJI-group23#4]
upload page has no css. gotta make it look good
<img width="838" alt="Screen Shot 2019-04-30 at 2 49 48 PM" src="https://user-images.githubusercontent.com/44102621/56989121-542be380-6b57-11e9-92f4-e19e2a9080b4.png">

**Comment by ElitaNelson** [comment]
class time 5.2.19

- trying to model this after how I did the UI for my databases project because that was aesthetically pleasing in my opinion

- originally (in previous project) had the html to assign a .php file to each button, but the upload php is still being updated, because we want it to work differently than it does rn (elaborated on in a separate issue), so I'm kind of working around that at the moment

- basically just copied my old html and am editing it as needed for now

**Comment by ElitaNelson** [comment]
I cannot for the life of me figure out how to make the upload button grey out after it's clicked and it feels absurd but i might blame it on being the end of finals week and being sick all week, including now
i think i made it more aesthetically pleasing though. it took me a minute to remember how to use HTML and CSS, it's been a minute

**Comment by AJM-1** [comment]
The button being grayed out is actually not as important now that the images are being assessed with ajax. The upload page does in fact look a lot more aesthetically pleasing, therefore I'm going to mark this as completed
